src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1999,7 +2062,60 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
     out.source.edge_target_set <+ new_edge_target;
 
 
+    // == Multi Connect
+    trace out.edge_creation_mode;
+
+    port_input_mouse_up_multi  <- inputs.hover_node_input.sample(&inputs.edge_multi_connect_up).unwrap();
+    port_output_mouse_up_multi <- inputs.hover_node_output.sample(&inputs.edge_multi_connect_up).unwrap();
```

Comment:
I don't believei ts correct. `inputs.hover_node_input` is for external control of the graph - like, backend can tell - hover this input. But this FRP endpoint should not be set by hovering the input with mouse. We should have a rule in FRP network that we never, ever emit events to `inputs` from inside of the FRP network and I think (hope) we do not do it here as well. The right solution here would be to use "outputs" instead of inputs, if our FRP network would set `outputs.hover_node_input` to something.